tests/helpers/acceptance.js

```diff
@@ -55,6 +58,18 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
+    .then(() => {
+      if (options.createESLintConfig) {
```

Comment:
We use `--skip-npm` on the target creation but we link in a unified PackageCache: https://github.com/ember-cli/ember-cli/blob/master/tests/acceptance/brocfile-smoke-test-slow.js#L34

... the `npm` assets will be there.